backend/catalogue/management/commands/generate_layer_previews.py

```diff
@@ -10,8 +10,9 @@
 
 from catalogue.models import Layer
 
+Image.MAX_IMAGE_PIXELS = None
 
-basemap = Image.open(default_storage.open('basemap.png'))
+basemap = Image.open(default_storage.open('land_shallow_topo_21600.tif'))
```

Comment:
Just an FYI, looks like this crept in.

Reply:
Hmm, yeah. I thought I had directly committed this on the develop branch as part of a separate ticket. Not sure if I accidentally committed on this branch, or whether things got a bit confused when I merged develop into this branch.